.github/workflows/test.yml

```diff
@@ -56,10 +63,10 @@ jobs:
       - name: Install poetry
         run: pipx install poetry
 
-      - name: Set up Python 3.11
-        uses: actions/setup-python@v4.5.0
+      - name: Set up Python
+        uses: actions/setup-python@v4.7.0
         with:
-          python-version: "3.11"
+          python-version-file: pyproject.toml
```

Comment:
Do you know if github actions parse python version from pyproject.toml? wow

Reply:
mmm but ou pyproject.toml still starts at 3.9
https://github.com/quipucords/quipucords/blob/main/pyproject.toml#L14